lib/metababel/bt2_stream_classes_generator.rb

```diff
@@ -203,6 +204,7 @@ def get_declarator(variable:, trace_class:, self_component:)
       pr "bt_stream_class_put_ref(#{variable});"
     end
 
+    # TODO: DEAD CODE, not needed any more, we used this with the old matching callbacks
```

Comment:
why do you mean?

Reply:
We should remove this code, but not in this PR